frontend/src/citizen-frontend/income-statements/IncomeStatementView.tsx

```diff
@@ -375,12 +399,15 @@ function Row({
       <Gap size="s" />
     </>
   )
-}
+})
 
-function makeYesNo(t: Translations) {
-  return (value: boolean): string => {
-    return value ? t.common.yes : t.common.no
-  }
+function useYesNo(t: Translations) {
+  return useCallback(
+    (value: boolean): string => {
+      return value ? t.common.yes : t.common.no
+    },
+    [t.common.yes, t.common.no]
+  )
```

Comment:
I think this would make more sense as a function in `Translations`. Something like `t.common.yesOrNo: (v: boolean) => string`. I think the `t.common.yes` and `t.common.no` keys are only used in these kinds of contexts anyway.

Reply:
Good idea, fixed!